content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+{{< image width="100%"
+    link="allocation.png"
+    caption="CPU and memory allocation and cost breakdown for sidecar and ambient scenarios"
+    >}}
+
+Going further, we can calculate a dollar amount for these numbers by referring to [GCP monthly pricing](https://cloud.google.com/compute/vm-instance-pricing) for a monthly cost per GB memory and per CPU. Consider these two different machines and their costs:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-4` (4 CPU, 32 GB) at $191/month
+
+Calculating the difference in memory shows 1GB costs $3.33/month. In our test this brings memory costs from $60 down to only $1 in the ztunnel-only scenario, and $6 with waypoints deployed.  Similarly for CPU:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-2` (2 CPU, 16 GB) at $95/month
+
+Calculating the difference in CPU shows 1 CPU costs $23/month. In our test this brings CPU costs from over $300 down to only $7 in the ztunnel-only scenario, and $34 with waypoints deployed.
+
+We can literally put a dollar amount on the savings ambient brings - and so can you!
+
+## Test It Out
+
+In comparison to what most users run Istio with in production, this test cluster is tiny. However, we expect even more savings with larger clusters and when more services are deployed. We encourage everyone to see what the savings with Istio ambient mesh looks like in their environments. Also note that these scripts have been pushed to GitHub so feel free to check them out [here](https://github.com/solo-io/ambient-performance/tree/boutique-demo). For tracking CPU and memory usage throughout the test scenarios, versions of [Prometheus](https://prometheus.io/), [node-exporter](https://prometheus.io/docs/guides/node-exporter/), and [Grafana](https://grafana.com/) are installed. A custom Grafana dashboard was created for observing relevant data, which can be found and imported from GitHub [here](https://github.com/solo-io/ambient-performance/blob/boutique-demo/dashboard/ambient-performance-analysis.json).
+
+## Conclusion
+
+Ka-ching. These results were collected with an alpha version of ambient, which is now [merged into the main branch](/blog/2023/ambient-merged-istio-main/) and included in Istio 1.18.
+
+Huge thanks to all the Istio maintainers who work very hard to make ambient as lean and performant as possible. Ambient service mesh’s goal of reducing infrastructure costs is bearing fruit and setting a solid foot forward on its roadmap to production readiness. These early numbers suggest users could cut their cloud usage by *99%* and resource requirements by *90%* – especially if users only require an L4 mesh.
+
```

Comment:
```suggestion
Our goals are to make ambient as lean and performant as possible. A second-order effect, though one that will be welcomed by many, is a reduction of infrastructure costs. These early numbers suggest users could cut their cloud usage by *99%* and resource requirements by *90%* – especially if users only require an L4 mesh.

```

a. you're writing this from the POV of an Istio maintainer
b. see comments above on if 99% is a realistic promise